CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -390,6 +407,9 @@ export default function Report({ reportID }) {
 
               <table className='w-full text-center mb-4 shadow-md'>
                 <thead className='bg-slate-100'>
+                <tr>
+                  <th colSpan={5} className="text-2xl">Report Statistics</th>
```

Comment:
Maybe you can add a bit of padding to all of the headers like, className="text-2xl p-2", just so that the heading stands out more and isn't as close to the border.